tasks/collections.go

```diff
@@ -1,31 +1,48 @@
 package tasks
 
+import "fmt"
+
 // Task 1: Write a function that creates and prints a slice of integers with values 1 to 5.
 
 func CreateAndPrintSlice() {
 	// Implement the code to create and print a slice of integers.
+	demo := []int{5, 7}
+	fmt.Println(demo)
 }
 
 // Task 2: Write a function that takes a slice of integers and calculates and prints their sum.
 
 func CalculateAndPrintSum(numbers []int) {
 	// Implement the code to calculate and print the sum of the numbers in the slice.
+	number := []int{6, 3}
+	var sum int = 0
+	for i := 0; i < 2; i++ {
+		sum += number[i]
+	}
+	fmt.Println(sum)
+
```

Comment:
No need to create a new slice number, you have to print the numbers slice already passed to the function. The test will fail because of this.